tests/unit/src/text/test_format.cpp

```diff
@@ -386,6 +386,235 @@ TEST_CASE( "Format::CargoString", "[Format][CargoString]") {
 	}
 }
 
+TEST_CASE( "Format::AmmoCount", "[Format][AmmoCount]") {
```

Comment:
This is missing a check for negative values, and another one for the scientific threshold.

Reply:
Negative ammo counts don't really have a meaning, so I don't think the return value with negative inputs really matters.